CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ endif()
 
 project(torch-mlir LANGUAGES CXX C)
 set(CMAKE_C_STANDARD 11)
```

Comment:
I did not apply the same change for the CMAKE_C_STANDARD , as downstream we use an older c version